index.js

```diff
@@ -111,10 +111,6 @@ File.prototype.clone = function(opt) {
   // Clone our file contents
   var contents;
   if (this.isStream()) {
-    if (typeof this.contents.clone !== 'function') {
```

Comment:
A couple of notes here:

a. the API compatible way to set `file._contents` is via `file.contents`. Are we supporting direct access to `_contents`? If so, yes, we should keep that check.
b. I've added a `cloneable.isCloneable(stream)` which is stricter.